### ISSUE TITLE
Shutdown is not working

### ISSUE THREAD

**Issue by JaeungHyun** [rationalmatter/Juno-Issue-Tracker#75]
*Environment (please complete the following information):**
 - Device: ipad pro 3rd 12.9
 - iOS version: ios 13.2
 - App version: latest

Shutdown button in the running tab is not working

**Comment by navoshta** [comment]
Hi @JaeungHyun — this seems to be working, could you specify what type of server are you experiencing this with: is it one of the cloud-computing services, or your own custom server? Also, perhaps it's your Jupyter server settings that don't immediately trigger list refresh, could you try tapping the Jupyter's refresh button (🔁)?

**Comment by JaeungHyun** [comment]
@navoshta nei, refresh button is also not working.

**Comment by mulra02** [comment]
Shutdown/refresh button in the ‘Running’ tab does not work for me as well. I can still shutdown from the ‘Files’ tab though.

**Comment by JaeungHyun** [comment]
@mulra02 Are you using own custom server? 
I'm using that.

**Comment by navoshta** [comment]
Hi @JaeungHyun, @mulra02 — I'm struggling to reproduce this unfortunately, could you please specify if you're both using a custom server or a cloud-computing service, and which Jupyter Notebook/JupyterLab version is installed on the server you are connecting to?

Either way, I'll look into what could be causing this on Juno Connect side — thanks for reporting.

**Comment by mulra02** [comment]
I’m also using my own custom server. Notebook version is 6.0.1.

**Comment by JaeungHyun** [comment]
@navoshta 

jupyter                   1.0.0                    py37_7
jupyter_client            5.3.4                    py37_0
jupyter_console           6.0.0                    py37_0
jupyter_core              4.6.0                    py37_0
jupyterlab                1.1.4              pyhf63ae98_0
jupyterlab_server         1.0.6                      py_0

I'm using my own custom server.

Ubuntu 18.04, install using conda

**Comment by navoshta** [comment]
Thanks @JaeungHyun, @mulra02 I will look into this. A dumb question perhaps, but this button does work when you connect to the same server on desktop, right?

**Comment by JaeungHyun** [comment]
@navoshta Yes, shutdown button is working when using desktop.  I'm using two servers, but same situation

**Comment by mulra02** [comment]
@navoshta You’re right, for me the button doesn’t work in desktop either. My issue is not caused by Juno then.
